system/settings/gbm.xml

```diff
@@ -47,6 +47,11 @@
         <setting id="videoscreen.screen">
           <visible>false</visible>
         </setting>
+	 <setting id="videoscreen.hwscalingfilter" type="boolean" label="39120" help="39121">
```

Comment:
indentation

Reply:
Looks like this is an error in the browser while viewing the diff in xml. I will confirm it on the git diff and do the changes accordingly.